src/App.tsx

```diff
@@ -16,35 +17,95 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType | ''>('');
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  enum SortType {
+    ALPHABET = 'alphabet',
+    LENGTH = 'length',
+  }
+
+  const visibleGoods = [...goodsFromServer].sort(
+    (good1: string, good2: string) => {
+      switch (sortField) {
+        case SortType.ALPHABET:
+          return good1.localeCompare(good2);
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    },
+  );
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
+  const handleReverse = () => {
```

Comment:
[CODE STYLE] - Function's name should start with a verb to clearly indicate what the method does. Consider renaming `handleReverse` to something like `reverseSortOrder` to enhance readability and intent.